### PR TITLE
Added --rdw to `download dataset` command to download variable-length dataset

### PR DESCRIPTION
**Pull request by std4lqi**
To support #58 

Signed-off-by: Qi Liang <31643673+std4lqi@users.noreply.github.com>

### CODE REVIEW

**Comment by dkelosky** [comment]
This seems to work for me 😄 .  In testing on windows, the file is downloaded with `x'0D0A'` carriage return & line feed.  These two bytes are not included in the `rdw`.

**Comment by lauren-li** [comment]
Thanks @std4lqi for adding this feature! Functionally, this PR seems to work for me. I am able to get RDW values in my downloaded data sets using the new `--rdw` option. There are a couple minor merge conflicts, but they should be able to be resolved without affecting the function of this PR.

The automated tests are mostly passing for me, but there may be a snapshot that needs to be updated:

Screen Shot 2021-04-27 at 1 20 57 PM

**Comment by std4lqi** [comment]
@lauren-li I updated snapshot for `--rdw`. Thanks!

**Comment by std4lqi** [comment]
@dkelosky Thanks for your testing! We use `rdw` heavily for binary variable length dataset. 

Usually, the EOL is appended in the dataset downloaded with `ASCII`, so even without `rdw`, we still can tell the end of record (text line in this case) with EOL. From my view, `TRANSFER_TYPE_ASCII_RDW` is not so useful, but I incline to adding for consistency, or there is some case I'm not aware of.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Latest Changes
```
Comment (lauren-li) [inline CHANGELOG.md:5]: @std4lqi Thanks for addressing my earlier comments! One more very minor request: Could you also please change "Latest" to "Recent" here so the Jenkins check can pass?

Screen Shot 2021-04-28 at 3 03 21 AM

Comment (std4lqi) [reply CHANGELOG.md]: I see it now, and update it. Thanks! :-)

**Review by lauren-li — APPROVED** [review]
This PR looks good to me now. Thanks @std4lqi for your hard work on this!

**Pull request merged**